tests/ci_test_list_1.txt

```diff
@@ -1,2 +1,3 @@
 tests/nn/misc/test_flatten_params_wrapper.py
 tests/nn/data_parallel/test_fsdp.py
+tests/nn/data_parallel/test_fsdp_conv.py
```

Comment:
I added this new test to catch known issues with pytorch version. I have seen py1.9 core dump on it.